motr/setup.c

```diff
@@ -1415,7 +1415,14 @@ static int be_repair_zone_pcnt_get(struct m0_reqh *reqh,
 			       &croot->rt_imeta_pver)) {
 			continue;
 		}
-		*repair_zone_pcnt = pver_obj->pv_u.subtree.pvs_attr.pa_K *
+		/**
+		 * Replaced K/P with S/P and safety coefficient 3/2 with 1/2
+		 * TODO: These changes should work fine as long as we
+		 * are using replication for DIX (i.e., N = 1). We need
+		 * to work on this formula for N > 1. Filed a Bug EOS-23309
+		 * to track it.
+		 */
+		*repair_zone_pcnt = pver_obj->pv_u.subtree.pvs_attr.pa_S *
```

Comment:
Let's assert here if repair spare percent crosses 25%, and user has to reduce spare or make it as zero.

Reply:
As discussed, got rid off safety coefficient. Now it asserts if repair zone percentage crosses 33%